src/metadata.rs

```diff
@@ -1,7 +1,8 @@
 use chrono::{DateTime, UTC};
-use crypto::digest::Digest;
-use crypto::ed25519;
-use crypto::sha2::Sha256;
+use ring;
```

Comment:
Pedantic: Can you move these to keep the imports in alphabetical order (just the crate, not the modules/structs).